src/sentry/discover/translation/mep_to_eap.py

```diff
@@ -75,7 +77,19 @@ def drop_unsupported_columns(columns):
         if column.startswith(COLUMNS_TO_DROP):
             dropped_columns.append(column)
         else:
-            final_columns.append(column)
+            should_drop = False
+            # regex for beginning of equation or function (if it starts with a parenthesis or comma)
+            regex_beginning = r".*[\(,]"
+            # regex for ending of equation or function (if it ends with a parenthesis or comma and other args)
+            regex_ending = r"((,.*)|\))"
+            # fields can be within the columns if the column is a function (like in count_if)
+            for field in FIELDS_TO_DROP:
+                if re.search(regex_beginning + re.escape(field) + regex_ending, column):
+                    dropped_columns.append(column)
+                    should_drop = True
+                    break
```

Comment:
Can we do `fields.parse_arguments` here instead and just loop through the arguments to see if they contain any fields to drop?